src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -523,14 +523,24 @@ public override void VisitForLoop(IForLoopOperation operation)
         public override void VisitForToLoop(IForToLoopOperation operation)
         {
             LogString(nameof(IForToLoopOperation));
-            LogLoopStatementHeader(operation);
+            LogLoopStatementHeader(operation, operation.IsChecked);
 
             Visit(operation.LoopControlVariable, "LoopControlVariable");
             Visit(operation.InitialValue, "InitialValue");
             Visit(operation.LimitValue, "LimitValue");
             Visit(operation.StepValue, "StepValue");
             Visit(operation.Body, "Body");
             VisitArray(operation.NextVariables, "NextVariables", logElementCount: true);
+
+            (ILocalSymbol loopObject, ForToLoopOperationUserDefinedInfo userDefinedInfo) = ((BaseForToLoopStatement)operation).Info;
+
+            if (userDefinedInfo != null)
+            {
+                _ = userDefinedInfo.Addition.Value;
+                _ = userDefinedInfo.Subtraction.Value;
+                _ = userDefinedInfo.LessThanOrEqual.Value;
+                _ = userDefinedInfo.GreaterThanOrEqual.Value;
+            }
```

Comment:
Consider adding a method to ForToLoopOperationUserDefinedInfo to force the evaluation of the lazies instances rather than enumerating the fields here. This would be more robust in the face of adding additional fields in the future.

Reply:
> Consider adding a method to ForToLoopOperationUserDefinedInfo to force the evaluation of the lazies instances rather than enumerating the fields here. This would be more robust in the face of adding additional fields in the future. 

I do not expect us to add anything else to that class in the future. I also think this method would look strange in the class because it is only useful as a test.

---
In reply to: [187120366](https://github.com/dotnet/roslyn/pull/26734#discussion_r187120366) [](ancestors = 187120366)